src/instructlab/lab.py

```diff
@@ -36,7 +35,52 @@
     import torch
 
 
-@click.group(cls=DYMGroup)
+class ExpandAliasesGroup(click.Group):
+    def __init__(self, *args, **kwargs):
+        self.aliases = kwargs.pop("aliases", {})
+        super().__init__(*args, **kwargs)
+
+    def get_command(self, ctx, cmd_name):
+        if cmd_name in self.aliases:
+            cmd = self.aliases[cmd_name]["cmd"]
+            print(
+                f"You are using an aliased command, this will be deprecated in a future release. Please consider using `ilab {group} {c}` instead"
+            )
```

Comment:
Is it possible to use `click.echo()` here and color it in red?

I would also adjust your wording here to say that this command is deprecated now and will be removed in a future release vs will be deprecated in the future.

Reply:
Also what are `{group}` and `{c}` ? It looks like they're not defined

Reply:
added back, I accidentally deleted these